source-notes/blog-litellm-auto-router-v2.md

```
@@ -0,0 +1,248 @@
+        adaptive: true
+
+        # optional: pin a session to its first-turn model (preserves prompt cache)
+        session_affinity: true
+        session_affinity_ttl_seconds: 3600
+
+      complexity_router_default_model: claude-sonnet-5
+```
+
+### Decision log line format ("Notes on the new pieces — Decision log" section, verbatim)
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)